### PR TITLE
change crate name to generic-token-faucet

### PR DESCRIPTION
**Pull request by throwbackjams**
Change instances of `devnet-token-faucet` to `generic-token-faucet`.

(note: changes in TS also include auto formatting on save)

### CODE REVIEW

**Review by jarry-xiao — APPROVED** [review]

File: tests/generic-token-faucet.ts [hunk 1]
```diff
@@ -1,42 +1,53 @@
...
+    .GenericTokenFaucet as Program<GenericTokenFaucet>;
 
-  let ticker = "AAA"
-  let ticker_lowercase = ticker.toLowerCase()
-  let decimals = 9
-  let amount_to_mint = 10 * 10^(decimals) 
+  let ticker = "AAA";
+  let ticker_lowercase = ticker.toLowerCase();
+  let decimals = 9;
+  let amount_to_mint = (10 * 10) ^ decimals;
```
Comment (jarry-xiao) [inline tests/generic-token-faucet.ts]: hold up, this is an XOR not an exponentiation right?

**Review by jarry-xiao — CHANGES_REQUESTED** [review]
See comment in test

File: tests/generic-token-faucet.ts (hunk 1 above)
Comment (throwbackjams) [reply tests/generic-token-faucet.ts]: Oof yeah fixed in latest commit [3253e64](https://github.com/Ellipsis-Labs/devnet-token-faucet/pull/2/commits/3253e64de2e002aa24194b326c4c5b14fd19482a)

**Review by jarry-xiao — APPROVED** [review]

**Review by seedasadsadgey — APPROVED** [review]

**Comment by seedasadsadgey** [comment]
> See comment in test

**Pull request merged**